### PR TITLE
add avatar css

### PR DESCRIPTION
**Pull request by lannisterrr**
https://user-images.githubusercontent.com/29068538/151650163-ea8bdaff-0023-4f20-bedf-a7c123405711.mp4

### CODE REVIEW

File: starterFiles/components/avatar.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
+@import url('../abstracts/variables.css');
+@import url('../base/reset.css');
+@import url('../utils/util.css');
+@import url('../utils/margin.css');
+@import url('../utils/padding.css');
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+
+/* avatar */
+.avatar-small {
```
Comment (Hamdrive) [inline starterFiles/components/avatar.css]: replace small, medium etc with shorter abbreviations (sm, md, lg, xl)

File: starterFiles/components/avatar.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
+@import url('../abstracts/variables.css');
+@import url('../base/reset.css');
+@import url('../utils/util.css');
+@import url('../utils/margin.css');
+@import url('../utils/padding.css');
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
```
Comment (shivampandey0) [inline starterFiles/components/avatar.css]: Don't need fonts for the component library as per the feedback I got yesterday.
You can add fonts to your documentation site when you make it.

File: starterFiles/components/avatar.css [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+/* avatar */
+.avatar-small {
+  width: 2rem;
+  height: 2rem;
+}
+
+.content-wrapper {
```
Comment (shivampandey0) [inline starterFiles/components/avatar.css]: could rename this class to `avatar-wrapper` to keep the consistency or put this class to some global file like helpers.

**Comment by aryan-13** [comment]
The code looks fine! Go ahead.

**Comment by lannisterrr** [comment]
Added the components working video

**Pull request merged**